src/ext/oblivc/obliv_bits.c

```diff
@@ -364,24 +364,22 @@ void protocolAddSizeCheck(ProtocolDesc* pd)
   t->cb.recv=sizeCheckRecv;
   t->cb.cleanup=sizeCheckCleanup;
 }
-// ---------------------------------------------------------------------------
+
+// --------------------------- Protocols -----------------------------------
+
+int ocCurrentParty() { return currentProto->currentParty(currentProto); }
+int ocCurrentPartyDefault(ProtocolDesc* pd) { return pd->thisParty; }
+
+ProtocolDesc* ocCurrentProto() { return currentProto; }
+void ocSetCurrentProto(ProtocolDesc* pd) { currentProto=pd; }
 
 void cleanupProtocol(ProtocolDesc* pd)
   { pd->trans->cleanup(pd->trans); }
 
 void setCurrentParty(ProtocolDesc* pd, int party)
```

Comment:
While at it, why not have this function be near ocSetCurrentProto two lines above :)